docs/guides/integrating-the-feature-hub.md

```diff
@@ -45,7 +45,8 @@ const {featureAppManager, featureServiceRegistry} = createFeatureHub(
 This creates the `FeatureServiceRegistry` singleton instance, registers all
 `featureServiceDefinitions` for the given integrator ID (`'acme:integrator'`),
 and instantiates a `FeatureAppManager` singleton instance using the
-`FeatureServiceRegistry`. Both singletons are returned.
+`FeatureServiceRegistry`. Both singletons are returned as properties of the
+`FeatureHub` instance created using the `createFeatureHub` function.
```

Comment:
we don't really talk about the `FeatureHub` itself elsewhere in the docs... Its just an interface bundling a bunch of stuff. Not an instance of some class.